neural_style.py

```diff
@@ -10,15 +10,25 @@
 import math
 from argparse import ArgumentParser
 
+from matplotlib import pyplot as plt
```

Comment:
Tried googling setting JPEG quality in various imsave functions, but unfortunately didn't find any, I've tried imsave from several packages (I think **scipy**, **PIL** and **matplotlib**), and first two seem to have JPEG quality 65 (or 75? do not remember which of the two) by default, and not changeable. The later one seems to have quality 95 also not changeable, but this is a lot better overall.